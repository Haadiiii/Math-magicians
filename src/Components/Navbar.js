import React from "react";
import { NavLink } from "react-router-dom";

const Navbar = () => (
  <div className="navbar">
    <h1>Math Magicians</h1>
    <nav>
      <ul>
        <li>
          <NavLink to="/">Home</NavLink>
        </li>
        <li>
          <NavLink to="/calculator">Calculator</NavLink>
        </li>
        <li>
          <NavLink to="/quote">Quote</NavLink>
        </li>
      </ul>
    </nav>
  </div>
);
export default Navbar;
