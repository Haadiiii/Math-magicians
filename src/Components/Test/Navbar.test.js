import React from "react";
import Navbar from "../Test/__mock__/Navbar";
import { render, cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";

afterEach(cleanup);

it("render Without crashing", () => {
  render(<Navbar />);
});

it("matches snapshot for Navbar", () => {
  const tree = renderer.create(<Navbar />).toJSON();
  expect(tree).toMatchSnapshot();
});
