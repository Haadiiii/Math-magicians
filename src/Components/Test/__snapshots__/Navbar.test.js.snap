// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for Navbar 1`] = `
<div
  className="navbar"
>
  <h1>
    Math Magicians
  </h1>
  <nav>
    <ul>
      <li>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          className=""
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          className=""
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
